### ISSUE TITLE
Rules within media queries duplicate when using multiple dimensions

### ISSUE THREAD

**Issue by Quazz** [addyosmani/critical#562]
Let's say we have a media query like so:

```
@media all and (min-width: 768px) {
.block {width: 100px;}

.title {width: 200px;}
}
```

Where .block may be seen on pages on certain resolutions in the critical viewpath but not others (which is the whole reason behind mutliple dimensions critical extraction of course). 

This will then generate multiple media queries in critical like so:
```
@media all and (min-width: 768px) {
.title {width: 200px;}
}
@media all and (min-width: 768px) {
.block {width: 100px;}

.title {width: 200px;}
}
(and more for more complicated rules and dimensions)
```

Thus increasing the critical file and potentially slightly messing up critical rendering (mostly impacting CLS).

**Comment by bezoerb** [comment]
@Quazz: sorry for the late response.
I added an option to customize cleancss settings and adjusted the defaults a bit so media queries should be merged now:
https://github.com/addyosmani/critical/pull/569

**Comment by bezoerb** [comment]
released in [v5.2.0](https://github.com/addyosmani/critical/releases/tag/v5.2.0)
